apps/auth/database.py

```
@@ -1,50 +1,16 @@
+try:
+	client = MongoClient(MONGO_URI,
+						maxPoolSize=50,
+						connectTimeoutMS=5000,
+						serverSelectionTimeoutMS=5000,
+						waitQueueTimeoutMS=5000)
+	# Verify connection
+	client.admin.command('ping')
+	db = client["wyra"] # This is the database name, you can't change it(for now, I need to talk to Zantex)
+	users = db["users"]
+except ConnectionFailure:
+	raise Exception("Failed to connect to MongoDB. Please check your connection settings.")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)